Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```
@@ -0,0 +1,6038 @@
+
+
+def find_dict_in_array(container: List[Dict[str, Any]], key: str, value: str) -> Optional[Dict[str, Any]]:
+    """Gets dictionary object in list of dictionaries.The search is by key that exist in each dictionary.
+
+    Args:
+        container (list): List of dictionaries.
+        key (str): Key to recognize the correct dictionary.
+        value (str]): The value for the key.
+
+    Returns:
+        Optional[Dict[str, Any]]: The dictionary / The dictionaries / None if there is no match.
+    """
+
+    for obj in container:
+        if obj.get(key) and str(obj[key]) == value:
+            return obj
+    return None
+
+
+def find_dicts_in_array(container: List[Dict[str, Any]], key: str, value: List[str]) -> Optional[List[Dict[str, Any]]]:
+    """Gets dictionaries object in list of dictionaries.The search is by key that exist in each dictionary.
+
+    Args:
+        container (list): List of dictionaries.
+        key (str): Key to recognize the correct dictionary.
+        value (List[str]): The values for the key.
+
+    Returns:
+        Optional[List[Dict[str, Any]]]: The dictionaries / None if there is no match.
+    """
+
+    return [obj for obj in container if obj[key] in value]
+
+
+def get_object_id(client: Client,
+                  create_response: Dict[str, Any],
+                  by_key: str,
+                  value: str,
+                  get_request: Callable,
+                  object_id: Optional[str] = None) -> str:
+    """Get object / sub object id. After create sub (member)
+        object we should get list of all members and get our id by some key.
+
+    Args:
+        client (Client): Fortiweb VM Client.
+        create_response (Dict[str, Any]): Response from create request.
+        by_key (str): Unique key to search the sub object.
+        value (str): The sub object value to the key.
+        get_request (Callable): Get request (for Fortiweb VM 1)
+        object_id (Optional[str]): Object ID (not sub object
```

```suggestion
```

This isn't neessarily a connection error. just let it be raised.

---

I don't understand. Can you clarify?